install/csm_installation_failure.md

```diff
@@ -0,0 +1,73 @@
+# CSM Installation Failure
+
+This page documents a possible timing related issue in a new CSM install.
+
+## Details
+
+<a name="verify_missing_secret"></a>
+### 1. Verify Missing Secret
+
+   1. Verify installation output signature matches
+
+      ```bash
+      ...
+        ERROR   Step: Set Management NCNs to use Unbound --- Checking Precondition
+      + Getting admin-client-auth secret
+      Error from server (NotFound): secrets "admin-client-auth" not found
+      + Obtaining access token
+      curl: (22) The requested URL returned error: 404
+      + Querying SLS
+      curl: (22) The requested URL returned error: 503
+
+      Check the doc below for troubleshooting:
+            If any management NCNs are missing from the output, take corrective action
+            before proceeding.
+
+       INFO  Failed Pipeline/Step id: f0cd9574240989eb04118d308f26b3ea
+      exit status 22
+      ```
+
+      Perform this procedure when observed. The root cause is simply a timing issue.
+
+<a name="validate_keycloak_setup_is_running"></a>
+### 1. Validate Keycloak Setup Is Running
```

Comment:
```suggestion
```

Reply:
[Still have this suggestion](https://github.com/Cray-HPE/docs-csm/pull/1717/files#r884192559)